src/main/java/uk/gov/hmcts/cmc/claimstore/events/claim/PostClaimOrchestrationHandler.java

```diff
@@ -65,12 +66,12 @@ public PostClaimOrchestrationHandler(
                 : claim;
 
         uploadClaimIssueReceiptOperation = (claim, authorisation, claimIssueReceipt) ->
-            claim.getClaimSubmissionOperationIndicators().getClaimIssueReceiptUpload() == NO
+            setDefaultIfNull(claim.getClaimSubmissionOperationIndicators().getClaimIssueReceiptUpload()) == NO
```

Comment:
I think we don't want to run the operation if it's null. But here we are doing opposite, if null we are considering it as `NO` and running the operation. Please double check